plugins/unmanaged_files/unmanaged_files_inspector.rb

```diff
@@ -90,7 +90,11 @@ def extract_unmanaged_files(files, trees, excluded, store_name)
     archive_path = File.join(store_path, "files.tgz")
     @system.create_archive(files.join("\0"), archive_path, excluded)
 
+    extracted_count = files.length
+
     trees.each do |tree|
+      phrase = Machinery::pluralize(extracted_count, "file or tree", "files and trees")
```

Comment:
I would not use an intermediate variable here. That makes it harder to see what the text would look like.